resources/ory/profile-production.yaml

```diff
@@ -30,10 +30,11 @@ hydra:
   deployment:
     resources:
       requests:
-        cpu: "250m"
-        memory: "256Mi"
+        cpu: 250m
```

Comment:
CPU limit is not set on purpose, we should remove our artificial CPU limit setting of 10 cores in runtime-overrides. Feel free to revert this change.